simcon_project/conversation_templates/tests/test_template_management.py

```diff
@@ -1,10 +1,11 @@
 from django.test import TestCase, RequestFactory
 from django.utils import timezone
 from django.shortcuts import reverse
-from users.models import Researcher
-from ..models import TemplateFolder, ConversationTemplate
+from datetime import date
+from users.models import Researcher, Student, Assignment, SubjectLabel
+from ..models import *
 from ..forms import FolderCreationForm
-from ..views.template_management import route_to_current_folder, remove_template, FolderEditView
+from ..views.template_management import RouteToCurrentFolder, FolderEditView
 
 
 class TemplateManagementTests(TestCase):
```

Comment:
What about the cleanup of the objects created in this suit? test_delete_template_cascades probably takes care of some or all of that, but I'm not sure. Also, I don't know anything about tests in Django haha

Reply:
I'll look into it and see how TestCase takes care of the objects that were created within the class

Reply:
So django creates a database specifically for testing then destroys it once the test is done so we don't have to worry about the objects that were created during the test